src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/SetAzureSqlDatabaseAuditingPolicy.cs

```diff
@@ -135,6 +144,10 @@ private void ApplyUserInputToBlobAuditingModel(DatabaseBlobAuditingPolicyModel m
                 model.AuditAction = AuditAction;
             }
 
+            if (EventType != null) // Event types are relevant only for Table auditing
+            {
+                throw new Exception(string.Format(Properties.Resources.EventTypeConfiguringIrrelevantForBlobAuditingPolicy));
```

Comment:
You could convert the AuditType to be a switch parameter (https://msdn.microsoft.com/en-us/library/dd878252(v=vs.85).aspx#Switch Parameters) in the cmdlet, then you could use parameter sets.